src/pagination/pagination.spec.ts

```diff
@@ -781,6 +949,21 @@ class TestComponent {
   maxSize = 0;
   ellipses = true;
   rotate = false;
+  FILTER_PAG_REGEX = /[^0-9]/g;
 
   onPageChange = () => {};
+  // User can apply the logic they want.
+  applyPage(myInputPage: HTMLInputElement) {
```

Comment:
1. I dont think there are tests that actually use these `applyPage` and `formatInput` functions
2. as said before, I would have extracted another test component for this as it is pretty custom